code/src/components/Header.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components';
+
+export const Header = () => {
+  return (
+    <Wrapper>
+      <h1>Macronutrient charts</h1>
+      <H3>Scan a product to see the calorie ratio </H3>
```

Comment:
Using styled components, why not give them a more descriptive name like `<Title>` and `<Subtitle>` or something 😎 Also, even though you're not styling your h1, I think it looks nicer when styled components are not mixed with html-tags, and maybe you want to style your h1 in the future (and then everything is already set up).

Reply:
Thanks for good input 😃Will fix!